### PR TITLE
Finished the design elements for Contact form

### PR DESCRIPTION
**Pull request by Medini-Rajendra**
## In this review, I have done the following changes
 - Finished the design elements required for contact form
 - Checked for linter errors. 
 - Form submit and input errors are validated.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,156 +22,139 @@
...
-            <div class="text15">Enter your message here...</div>
-        </section>
-        <!--section class="rectangle18"!-->
-        <a href="#" class="border-button_startcolab">Start collaboration</a>
-            <!--div class="text16">Start collaboration</div!-->
-        <!--/section!-->
-        <section class="rectangle19">
+        <section class="Contact-form">
+            <img class="img11" src="left_topside_icon.svg" alt="Image geometry on the left top side">
+            <form name="Contactform" action="https://formspree.io/f/xbjqawyz" onsubmit="return validateForm()" method="post" aria-required>
```
Comment (vichuge) [inline index.html]: - [ ]  Try to use `POST` in place of `post` (uppercase). 🐶

File: index.html [hunk 2]
```diff
@@ -13,156 +22,139 @@
...
+                <h1 class="text10">Get started</h1>
+                <p class="contact-form-para">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+                <ul class="contact-form-ullist">
+                    <li class="contact-form-name">
+                        <input type="text" maxlength="30" class="nooutline" id="name" name="user_name" placeholder="Your name">
```
Comment (vichuge) [inline index.html]: - [ ] The inputs inside the form should have the attribute `required`. This helps to validate a value on input with HTML5 help. It's easier than using an alert. Just add `required` inside the input tag. 🐱

File: index.html [hunk 3]
```diff
@@ -13,156 +22,139 @@
...
+                        <!--div class="sample_text">Anastasiia</div!-->
+                    </li>
+                    <li class="contact-form-email"><input type="email" id="mail" class="nooutline" name="user_email" placeholder="Your email"></li>
```
Comment (vichuge) [inline index.html]: - [ ] The inputs inside the form should have the attribute `required`. This helps to validate a value on input with HTML5 help. It's easier than using an alert. Just add `required` inside the input tag. 🐱

File: index.html [hunk 4]
```diff
@@ -13,156 +22,139 @@
...
+                    <li class="contact-form-message"><textarea maxlength="500" id="msg" class="nooutline" name="user_message" placeholder="Enter your message here..."></textarea></li>
```
Comment (vichuge) [inline index.html]: - [ ] The inputs inside the form should have the attribute `required`. This helps to validate a value on input with HTML5 help. It's easier than using an alert. Just add `required` inside the input tag. 🐱

File: index.html [hunk 5]
```diff
@@ -5,6 +5,15 @@
...
         <link rel="stylesheet" href="style.css" type="text/css">
+        <script>
+            function validateForm() {
+                var x = document.forms["Contactform"]["user_name"].value;
+                if (x == "" || x == null) {
+                    alert("Name must be filled out");
+                    return false;
+                }
+            }
+        </script>
```
Comment (vichuge) [inline index.html]: - [ ] This is not necessary when you add `required` on your inputs. 🌵

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [ ] Check the comments under the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Medini-Rajendra) [reply index.html]: I have done the requested change.

File: index.html (hunk 2 above)
Comment (Medini-Rajendra) [reply index.html]: I have put the required in the input tag.

File: index.html (hunk 3 above)
Comment (Medini-Rajendra) [reply index.html]: I have put the required in the input tag.

File: index.html (hunk 4 above)
Comment (Medini-Rajendra) [reply index.html]: I have put the required in the input tag.

File: index.html (hunk 5 above)
Comment (Medini-Rajendra) [reply index.html]: I have removed the above snippet from the HTML file

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi @Medini-Rajendra, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**